packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -123,6 +120,12 @@ interface MongoNavInterface {
    */
   className?: string;
 
+  /**
+   * Click EventHandler that receives a `type` as its first argument and the associated `MouseEvent` as its second
+   * This prop provides a hook into product link and logout link clicks and allows consuming applications to handle routing internally
+   */
+  onElementClick?: (type?: OnElementClick, event?: React.MouseEvent) => void;
```

Comment:
Are these arguments optional? Aren't they always going to be there?

Reply:
👍